packages/tools/devtools/devtools-view/src/components/data-visualization/EditableView.tsx

```diff
@@ -0,0 +1,335 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+
+import {
+	Button,
+	Dropdown,
+	DropdownProps,
+	Input,
+	InputOnChangeData,
+	Option,
+	tokens,
+} from "@fluentui/react-components";
+import {
+	DataEdit,
+	EditType,
+	FluidObjectValueNode,
+	Primitive,
+} from "@fluid-experimental/devtools-core";
+
+import { Serializable } from "@fluidframework/datastore-definitions";
+import { useMessageRelay } from "../../MessageRelayContext";
+import { TreeHeader } from "./TreeHeader";
+
+/**
+ * Input to {@link EditableView}
+ */
+export interface EditableViewProps {
+	node: FluidObjectValueNode;
+	containerKey: string;
+	label: string;
+}
+
+/**
+ * Input to EditableComponents
+ */
+interface EditableComponentProps {
+	node: FluidObjectValueNode;
+	setEditing: React.Dispatch<React.SetStateAction<boolean>>;
+	submitChange: (data: Serializable<unknown> | Primitive) => void;
```

Comment:
```suggestion
	submitChange: (data: Serializable<unknown> | null | undefined) => void;
```

Reply:
There are a couple of other places still using `| Primitive` too.

Reply:
I think I got rid of them